src/Templates/src/templates/maui-blazor/MauiProgram.cs

```diff
@@ -16,9 +16,11 @@ public static MauiApp CreateMauiApp()
 			});
 
 		builder.Services.AddMauiBlazorWebView();
+//-:cnd:noEmit
 #if DEBUG
 		builder.Services.AddBlazorWebViewDeveloperTools();
 #endif
+//+:cnd:noEmit
```

Comment:
This change is unrelated but I found it when I was testing something else using the Blazor template. The two additions prevent the directives from being processed by the templating engine.